File: Activity41.c
#include <stdio.h>

/* Function prototype */
int getArea(int length, int width);

/* Global variable to store result */
int area;

int main() {
    int length, width;

    printf("WELCOME TO NICO'S COMPUTING AREA OF THE RECTANGLE PROGRAM\n\n");

    /* Ask user input */
    printf("Input length: ");
    scanf("%d", &length);

    printf("Input width: ");
    scanf("%d", &width);

    /* Call function */
    area = getArea(length, width);

    /* Display result */
    printf("Output: the area is : %d\n", area);

    return 0;
}

/* Function definition */
int getArea(int length, int width) {
    return length * width;
}

File: Java/Activity41.java
import java.util.Scanner;

public class Activity41 {

    public static float getArea(float length, float width) {
        return length * width;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("WELCOME TO NICO’S COMPUTING AREA OF THE RECTANGLE PROGRAM");

        System.out.print("Input length: ");
        float length = scanner.nextFloat();

        System.out.print("Input width: ");
        float width = scanner.nextFloat();

        float area = getArea(length, width);

        System.out.printf("Output: the area is : %.2f%n", area);

        scanner.close();
    }
}
